core/src/main/java/com/opensymphony/xwork2/ognl/OgnlValueStackFactory.java

```diff
@@ -125,6 +125,7 @@ protected boolean containerAllowsStaticMethodAccess() {
 
     /**
      * Retrieve allowStaticFieldAccess state from the container (allows for lazy fetching)
+     * Note: Defaults to true when no state is defined in the container (same default behaviour as OgnlUtil)
```

Comment:
Now this should be vanished  as well :)

Reply:
Sorry ... I missed that.  The latest commit removes the (now) unwanted comment line.  Thanks for catching that.  :)